src/utils/read-pacts.ts

```diff
@@ -2,19 +2,40 @@ import * as fs from 'fs';
 import * as path from 'path';
 
 export function readPacts(dir: string): Pact[] {
-    const fileNames = fs.readdirSync(dir);
+    const filePaths = fs
+        .readdirSync(dir)
+        .map((fileName) => path.join(dir, fileName));
```

Comment:
Do we need concatenate "dir" with "fineName" here ? In previous version of code that was not necessary, and the information about "dir" is only needed in error string ?

I wouldn't do that concatenation here, and use `dir` directly in string template:
```js
`Pact interactions for provider: ${pact.provider?.name} in pact file ${dir}${fileName} are empty.
```